lib/expansion_rules.rb

```diff
@@ -83,6 +83,7 @@ def details_fields(*fields)
   TAXON_FIELDS = (DEFAULT_FIELDS + %i[description details phase]).freeze
   NEED_FIELDS = (DEFAULT_FIELDS + details_fields(:role, :goal, :benefit, :met_when, :justifications)).freeze
   FINDER_FIELDS = (DEFAULT_FIELDS + details_fields(:facets)).freeze
+  MINISTERIAL_ROLE_FIELDS = (DEFAULT_FIELDS + details_fields(:body, :seniority, :role_payment_type)).freeze
```

Comment:
I think it would be good if this was defined as:

```suggestion
  MINISTERIAL_ROLE_FIELDS = (ROLE_FIELDS + details_fields(:seniority)).freeze
```

To ensure that if we add fields to the other roles, it doesn't accidentally get forgotten from this list.